daffodil-test/src/test/resources/org/apache/daffodil/section00/general/tunables.xml

```diff
@@ -17,7 +17,9 @@
 -->
 
 <daf:dfdlConfig
-	xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext">
+	xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext org/apache/daffodil/xsd/dafext.xsd">
```

Comment:
This xsi:schemaLocation must not be running into Daffodil-2339 bug because of the way it is used in tests. A different test dies on this same xsi:schemaLocation being loaded by the CLI. This one must be used from TDML runner, which may not be validating it?